intelhex/__init__.py

```diff
@@ -544,14 +544,19 @@ def _get_eol_textfile(eolstyle, platform):
             raise ValueError("wrong eolstyle %s" % repr(eolstyle))
     _get_eol_textfile = staticmethod(_get_eol_textfile)
 
-    def write_hex_file(self, f, write_start_addr=True, eolstyle='native', byte_count=16):
+    def write_hex_file(self, f, write_start_addr=True, ext_addr_mode='linear', eolstyle='native', byte_count=16):
```

Comment:
don't add your new parameter in the middle of the existing parameters, put it to the end.

Reply:
I usually add new parameters at the end, but I thought this will be more natural/readable. I will change this in next 'push'.